virttest/qemu_vm.py

```diff
@@ -1043,7 +1043,19 @@ def set_value(opt_string, key, fallback=None):
             elif optget("spice_port") != "no":
                 set_value("port=%s", "spice_port")
 
-            set_value("password=%s", "spice_password", "disable-ticketing=on")
+            password = optget("spice_password")
+            secret_cmdline = ''
+            if password:
+                if ("password-secret" not in
+                        devices.execute_qemu("-spice help")):
+                    spice_opts.append('password=%s' % password)
+                else:
+                    secret_id = 'spice_sec0'
+                    secret_cmdline = devices.secret_object(
```

Comment:
Missed,  `secret_object_define_by_varibles`

Reply:
@yanan-fu Ops, failed to add the change, just updated, thanks.